votor/src/alpenglow_metrics.rs

```diff
@@ -0,0 +1,116 @@
+use {
+    histogram::Histogram,
+    solana_pubkey::Pubkey,
+    solana_votor_messages::vote::Vote,
+    std::{collections::BTreeMap, time::Duration},
+};
+
+#[derive(Debug)]
+struct VoteMetrics {
+    notar: (u64, Histogram),
+    notar_fallback: (u64, Histogram),
+    skip: (u64, Histogram),
+    skip_fallback: (u64, Histogram),
+    final_: (u64, Histogram),
+}
+
+impl VoteMetrics {
+    fn new() -> Self {
+        // TODO: pick sensible parameters for the buckets
+        let histogram = Histogram::new(10, 10).unwrap();
+        Self {
+            notar: (0, histogram.clone()),
```

Comment:
struct here might be good to avoid using tuple notation elsewhere

Reply:
switched to struct.